content/en/docs/reference/kubectl/jsonpath.md

```diff
@@ -98,4 +98,17 @@ kubectl get pods -o=jsonpath="{range .items[*]}{.metadata.name}{\"\t\"}{.status.
 ```
 {{< /note >}}
 
+{{< note >}}
+
+JSONPath regular expressions are not supported. If you want to match using regular expressions, you can use a tool such as `jq`.
+
+```shell
+# kubectl does not suport regular expressions
```

Comment:
```suggestion
# `kubectl -o jsonpath` does not suport regular expressions
```

Reply:
You don't need backticks in comments (shell scripts are not Markdown).

Reply:
Yeah, I know it's not Markdown, I personally use them in code comments just for visual separation. Otherwise it feels like a bit of a "[garden path](https://en.wikipedia.org/wiki/Garden-path_sentence)" to me, but that might be idiosyncratic.

Reply:
I'd use single quotes or double quotes; we shouldn't assume readers are familiar with Markdown and its conventions.
Ah, and I just spotted something else, a typo.
```suggestion
# kubectl does not support regular expressions for JSONpath output
```